Makefile

```diff
@@ -281,7 +281,7 @@ deploy/openshift/deploy-kiali-to-openshift.sh
 ## openshift-undeploy: Undeploy from Openshift project.
 openshift-undeploy: .openshift-validate
 	@echo Undeploying from OpenShift project ${NAMESPACE}
-	${OC} delete all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,virtualservices,destinationrules,ingresses --selector=app=kiali -n ${NAMESPACE}
+	${OC} delete all,secrets,sa,templates,configmaps,deployments,clusterroles,clusterrolebindings,virtualservices,destinationrules,ingresses,customresourcedefinitions --selector=app=kiali -n ${NAMESPACE}
```

Comment:
it's different, here with `customresourcedefinitions` I'm asking to remove the *definitions* (the CRDs), not the custom objects themselves. So it's still necessary to explicitly remove virtualservices and destinationrules

Reply:
(however the first item we delete is "all", so I wonder why we need all the others)

Reply:
"all" never has been _all_, but only a small subset

Reply:
Ok so "customresourcedefinitions" can delete all definitions from the selector, so, without selector it can delete all definitions from the given namespace.

Reply:
@lucasponce I guess yes

Reply:
will this delete all objects of the kind:

```
apiVersion: "monitoring.kiali.io/v1alpha1"
kind: MonitoringDashboard
```
??

(assuming they all have an app label - which our vert.x one does not - see my previous comment about that)

Reply:
no (see response on previous comment)